src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractReferenceFinder.cs

```diff
@@ -316,15 +312,15 @@ public static ReferenceLocation CreateReferenceLocation(FindReferencesDocumentSt
     {
         foreach (var aliasSymbol in localAliasSymbols)
```

Comment:
>foreach (var aliasSymbol in localAliasSymbols) [](http://example.com/codeflow?start=8&length=46)

Should we be checking the cancellationToken anywhere in these newly sync methods?

Reply:
nevermind, I see they still throw down in the bowels